app/Jobs/SendMessageToChatwork.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\Bot;
+use App\Models\MessageHistory;
+use Illuminate\Bus\Queueable;
+use SunAsterisk\Chatwork\Chatwork;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use SunAsterisk\Chatwork\Exceptions\APIException;
+
+class SendMessageToChatwork implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    protected $bot;
+    protected $room_id;
+    protected $messages;
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Bot $bot, $room_id, $messages)
+    {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Jobs/SendMessageToChatwork.php</br>**Line**: 30</br>Variable "room_id" is not in valid camel caps format